packages/thunderstore-api/src/get/__tests__/teamMembers.test.ts

```diff
@@ -10,5 +10,5 @@ it("ensures accessing team members requires authentication", async () => {
       data: {},
       queryParams: {},
     })
-  ).rejects.toThrowError("401: Unauthorized");
+  ).rejects.toThrowError("Authentication required. Please sign in.");
```

Comment:
If the thrown error gives us clean access to the status code, it could be less brittle to check that instead of the message. Unless the message itself is the very behaviour we are interested in.